docs/release-notes/november-2023.md

```diff
@@ -65,21 +66,21 @@ If no issues are detected after a few days, the Semgrep team then promotes the `
 - **CLI tool:**
     - The `--severity=[VALUE]` option, which can be added to a `semgrep scan` command, has been fixed. <!-- gh-9062 -->
     - The `--sarif` flag no longer crashes when Semgrep itself encounters errors.
-- Semgrep now refuses to run incompatible versions of the Pro Engine, rather than crashing with a confusing error message. <!-- (gh-8873) -->
+- Semgrep now refuses to run incompatible versions of the Pro Engine, rather than crashing and returning a confusing error message. <!-- (gh-8873) -->
 - Fixed an issue where the CI provider icons disappeared from the **Scan new project in CI** window. The icons now appear. <!-- 11228 -->
 - Implemented minor fixes for the new onboarding flow. <!-- 11209, 11207 -->
 
 ## 💻 Semgrep Code
 
 ### Added
 
-- **Findings details page:** For users that have granted Semgrep code access, you are now able to view **in-app code snippets** in Semgrep Cloud Platform.
+- **Findings details page:** For users that have granted code access to Semgrep, you can now view **in-app code snippets** in the Semgrep Cloud Platform.
```

Comment:
Another interesting thing here is that Semgrep Cloud Platform is a proper noun, like Jason or Grindr, and we treat it as such. (We had a long conversation that wound up with this decision, in which we talked about how it's a proper noun made up of common nouns which is what makes it sound "wrong" without the "the" but also it's an actual product with a trademark.)

[https://www.notion.so/semgrep/Semgrep-technical-communication-style-guide-4d1104e78927460aa8c7624aedd6561f?pvs=4#64ffc68b6123419aa3bfa444a1f26461](https://www.notion.so/semgrep/Semgrep-technical-communication-style-guide-4d1104e78927460aa8c7624aedd6561f?pvs=4#64ffc68b6123419aa3bfa444a1f26461)

I'll add in the rest of the suggestion.